src/pages/search.js

```diff
@@ -39,18 +38,13 @@ export default function Search() {
       const data = await response.json();
       const cities = data.results;
       setCitiesData(cities);
-      // console.log('cities inside the function:', cities);
     } catch (err) {
       console.log(err);
```

Comment:
I checked for response because error messages received within a response from the server (like 404 or 500 for instance) do not register automatically as an error, because the promise resolves,  and do not show up in the .catch method.